lib/src/base-styles/_nx-globals.scss

```diff
@@ -18,19 +18,44 @@ $-nx-scrollable-default-height: 400px;
   color: var(--nx-color-link);
   cursor: pointer;
 
-  &:hover {
-    color: var(--nx-color-link-hover);
+  .nx-code {
+    color: var(--nx-color-link);
   }
 
-  &:active {
-    color: var(--nx-color-link-active);
+  &:hover {
+    &, .nx-code {
+      color: var(--nx-color-link-hover);
+    }
   }
 
   &:focus {
-    border-radius: var(--nx-border-radius);
+    // outline-offset scales the border-radius so this
+    // was changed to 1px to get it visually close to
+    // var(--nx-border-radius) or (6px).
+    border-radius: 1px;
     color: var(--nx-color-link-focus);
     outline: 1px solid var(--nx-color-interactive-border-focus);
-    outline-offset: 2px;
+    outline-offset: 1px;
+
+    .nx-code {
+      color: var(--nx-color-link-focus);
+    }
+  }
+
+  &:active {
+    color: var(--nx-color-link-active);
+
+    .nx-code {
+      color: var(--nx-color-link-active);
```

Comment:
Can use the `&, .nx-code {` pattern in these two places as well

Reply:
> Can use the `&, .nx-code {` pattern in these two places as well

Cleaned it up here: abda55ee13b4fb3a177f2b74d570d49eb66e59e3